README
A general Blood bank management system


ChangeLog
commit 5f4a031f3b2b93e2632c41db1aead309352fdfc7
Author: Anish A <aneesh.nl@gmail.com>
Date:   Sun Jul 4 14:45:38 2010 +0530

    Hashed password complete

commit 1dc3ed651dba1efce07bfb9c364d792fadd52dc4
Author: Anish A <aneesh.nl@gmail.com>
Date:   Sun Jul 4 14:33:40 2010 +0530

    Changed hasing function

commit 84204fc4103696ba54d8582ddd3bb5cc5d8d163e
Author: Aneesh A <aneesh@ACS-SUPER-COMPUTER.(none)>
Date:   Sat Jul 3 23:42:19 2010 +0530

    Added gitignore to ignore cnn.php and netbeans project files

commit 87acdff0c626b7266655da388d6eed8ceb9ad915
Merge: 941a3a6 5a55eea
Author: aneesh <aneesh.nl@gmail.com>
Date:   Thu Jul 1 00:27:02 2010 +0530

    Merge branch 'installer'
    
    Conflicts:
    	install.php

commit 5a55eeaee40161476d97e92904912ce51c3f2a98
Author: aneesh <aneesh.nl@gmail.com>
Date:   Tue Jun 29 23:26:11 2010 +0530

    Updated TODO

commit 69c197b517c572911dc8bce94164252252920325
Author: aneesh <aneesh.nl@gmail.com>
Date:   Tue Jun 29 23:25:06 2010 +0530

    Finished Installer

commit f55eafa920ff8607dc77c5cd7d4b98517d5e2ea3
Author: aneesh <aneesh.nl@gmail.com>
Date:   Tue Jun 29 20:00:31 2010 +0530

    Connection File generation ok. Continuing with database

commit 8833320febf335a672cd2387a0853aabe3221be5
Author: aneesh <aneesh.nl@gmail.com>
Date:   Mon Jun 28 09:59:07 2010 +0530

    Organized inputs

commit 6b830cf6bf1b9405da68a85148f5294ec03a1aed
Author: aneesh <aneesh.nl@gmail.com>
Date:   Mon Jun 28 07:38:57 2010 +0530

    Added some Feilds : User Interface OK

commit 809439ac0c9dceba0f01d9eea1b10f6429fb1db5
Author: aneesh <aneesh.nl@gmail.com>
Date:   Mon Jun 28 07:37:58 2010 +0530

    Code cleanup

commit 0ac0c2e8ec338ccc92d5ab3c7ca85183dee491a2
Author: aneesh <aneesh.nl@gmail.com>
Date:   Sun Jun 27 23:16:56 2010 +0530

    Switched to new branch for creating installer

commit 941a3a657e42f94467b990c18cb10bc73ac06a57
Author: aneesh <aneesh.nl@gmail.com>
Date:   Sun Jun 27 20:09:30 2010 +0530

    Installer work : User interface design in progress

commit 11fcd90c92accc89df3a5d037aaebb0a3fe1baf9
Author: aneesh <aneesh.nl@gmail.com>
Date:   Sun Jun 27 20:09:05 2010 +0530

    Installer work undertaken by Anish A

commit d547a60b23ffa92646be9c520b3dfb1cea1e5c91
Author: aneesh <aneesh.nl@gmail.com>
Date:   Sun Jun 27 19:04:51 2010 +0530

    Updated TODO and undertaken some.

commit 8cc04cf264bc1d748f0248c20ca8d371f0b5d2c3
Author: aneesh <aneesh.nl@gmail.com>
Date:   Sun Jun 27 00:56:02 2010 +0530

    Added updated database schema with sample data

commit 76a66eda4b71b6564aed81dcc826337c378dedd0
Author: aneesh <aneesh.nl@gmail.com>
Date:   Sun Jun 27 00:53:43 2010 +0530

    Updated About Us

commit 4f828114bfd902000906dfb0ee4eedf1205f5231
Author: aneesh <aneesh.nl@gmail.com>
Date:   Sun Jun 27 00:47:36 2010 +0530

    Updated Changelog

commit 7ab8248bf28f33b140ed28b4a0746c463e6b0f99
Author: aneesh <aneesh.nl@gmail.com>
Date:   Sun Jun 27 00:47:01 2010 +0530

    Fixed alignment problems

commit 927dcc2c1f578ab29108635767a05757e29f464a
Author: aneesh <aneesh.nl@gmail.com>
Date:   Sun Jun 27 00:09:58 2010 +0530

    Some colour gimmics in profile

commit 7d2dfd0cdca3a1974ba1abc78df18f5499c1e70e
Author: aneesh <aneesh.nl@gmail.com>
Date:   Sat Jun 26 23:57:27 2010 +0530

    Added drop down lists for date of birth and last date of donation selection

commit 55a176aaca1bb54c012ab2bf4fdc81319573c99e
Author: aneesh <aneesh.nl@gmail.com>
Date:   Sat Jun 26 23:55:47 2010 +0530

    Added Function to split date to components

commit ab275d417922f925be6f593567755a19492aee70
Author: aneesh <aneesh.nl@gmail.com>
Date:   Sat Jun 26 23:23:57 2010 +0530

    Changed Age to Date of birth

commit 32f2e84df41ab34b35e0603613578f05c27ed9b0
Author: aneesh <aneesh.nl@gmail.com>
Date:   Sat Jun 26 22:48:56 2010 +0530

    Alignment problem rectified. Also Changed to Date drop down box

commit 1a3ef696b9f58f7441671b0c6868e22a892f8aee
Author: aneesh <aneesh.nl@gmail.com>
Date:   Sat Jun 26 22:29:30 2010 +0530

    Unstable changes

commit 60a3433b6422eab6dad070be4c592513c22e6334
Author: aneesh <aneesh.nl@gmail.com>
Date:   Sat Jun 26 22:28:34 2010 +0530

    Added ChangeLog, Automatically generated by git

commit 5ef1a28b973e15e512eacc8d3cba7b9ad2f145a2
Author: aneesh <aneesh.nl@gmail.com>
Date:   Sat Jun 26 22:27:51 2010 +0530

    Work of Date of birth problem undertaken by Anish A

commit f5f66e6f149f880c9485f2c992db8298f4c86357
Author: aneesh <aneesh.nl@gmail.com>
Date:   Sat Jun 26 22:26:21 2010 +0530

    Minor changes

commit da1ee8ef6922398703c456efc465a1d604117004
Author: aneesh <aneesh.nl@gmail.com>
Date:   Sat Jun 26 22:24:57 2010 +0530

    Added some functions to calculate class file

commit 8fb9cfd25cd90136a0b19b18ba43ac0374eb9ab3
Author: aneesh <aneesh.nl@gmail.com>
Date:   Fri Jun 25 14:25:06 2010 +0530

     Updated TODO

commit 7cff9ce3ed76f399039951f047a7d6e54237235c
Author: aneesh <aneesh.nl@gmail.com>
Date:   Fri Jun 25 14:24:25 2010 +0530

     Linked public profile to the names

commit a0e1d5773fc576795b50f7d6ab63a3fe0a5cb707
Author: aneesh <aneesh.nl@gmail.com>
Date:   Fri Jun 25 14:15:26 2010 +0530

     Minor changes

commit b9904c44bf17fe885452893f1f8e24c5978c5f3b
Author: aneesh <aneesh.nl@gmail.com>
Date:   Fri Jun 25 14:07:38 2010 +0530

    Public Profile engine ok

commit 8e12d4c0fec8897d6aefc65c8090bd38d00a944d
Author: aneesh <aneesh.nl@gmail.com>
Date:   Fri Jun 25 12:39:08 2010 +0530

    Remove inconsistency in the user interface of edit profile : Fixed

commit 4908fefb606363f2985a19026407b712ca98e484
Author: aneesh <aneesh.nl@gmail.com>
Date:   Fri Jun 25 10:53:02 2010 +0530

     Added empty file

commit 16160acbafef258cedfc0a2434abe88810642ff9
Author: aneesh <aneesh.nl@gmail.com>
Date:   Fri Jun 25 10:52:31 2010 +0530

    Changed TODO and removed datepicker

commit aa4366937926197b5e73f76b914eb53e171f6413
Author: aneesh <aneesh.nl@gmail.com>
Date:   Sat Jun 26 04:48:15 2010 +0530

    Disabled registrations temporarly


reg.php
<?php ob_start();

 ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//E N" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>Register a Donor</title>

<script type="text/javascript">
<!--
function MM_jumpMenu(targ,selObj,restore){ //v3.0
  eval(targ+".location='"+selObj.options[selObj.selectedIndex].value+"'");
  if (restore) selObj.selectedIndex=0;
}
//-->
</script>
<style type="text/css">
<!--
#header {
    font-size: 150pt;
    font-weight: bold;
    color: #000000;
}
-->
</style>
</head>

<body>
<script type="text/javascript">
function validate(form_id,email) {
   var reg = /^([A-Za-z0-9_\-\.])+\@([A-Za-z0-9_\-\.])+\.([A-Za-z]{2,4})$/;
   var address = document.forms[form_id].elements[email].value;
   if(reg.test(address) == false) {
      alert('Invalid Email Address');
      return false;
   }
}
function check() {
   var reg = /^([A-Za-z0-9_\-\.])+\@([A-Za-z0-9_\-\.])+\.([A-Za-z]{2,4})$/;
   var address = document.formcheck.email.value;
   if(reg.test(address) == false) {
      alert('Invalid Email Address');
      return false;
   }
    if(document.formcheck.name.value=="") {
        alert("Enter name");
        document.formcheck.name.focus();
        return false;
    }
    if(document.formcheck.Regdate.value=="") {
        alert("Enter Your Age");
        document.formcheck.Regdate.focus();
        return false;
    }
    if(document.formcheck.Regdate.value < 18) {
        alert("You must be above 18 to donate blood.");
        document.formcheck.Regdate.focus();
        return false;
    }
    if(document.formcheck.classs.value=="") {
        alert("Select the class");
        document.formcheck.quantity.focus();
        return false;
    }
    if(document.formcheck.jumpMenu.selectedIndex==0) {
        alert("Select Your Blood Group");
        document.formcheck.jumpMenu.focus();
        return false;
    }
    if(document.formcheck.weight.value=="") {
        alert("Enter Weight");
        document.formcheck.weight.focus();
        return false;
    }
    if(document.formcheck.last.value=="") {
        alert("Enter Last Donated Date(Put 0 if never donate)");
        document.formcheck.last.focus();
        return false;
    }
    if(document.formcheck.phone.value=="") {
        alert("Enter Contact Phone Number");
        document.formcheck.phone.focus();
        return false;
    }
    if(document.formcheck.email.value=="") {
        alert("Enter Email");
        document.formcheck.email.focus();
        return false;
    }
    if(document.formcheck.password.value=="") {
        alert("Enter Password");
        document.formcheck.password.focus();
        return false;
    }
    if(document.formcheck.repass.value=="") {
        alert("Re-enter The Same Password Given Above");
        document.formcheck.repass.focus();
        return false;
    }
    if(document.formcheck.repass.value !== document.formcheck.password.value) {
        alert("Password Mismatch Please Check It");
        document.formcheck.repass.focus();
        return false;
    }
    if(document.formcheck.address3.value=="Select") {
        alert("Select District");
        document.formcheck.address3.focus();
        return false;
    }
}
</script>


<table width="100%" border="0" cellspacing="0" cellpadding="0">
  <tr>
    <td><table width="900" border="1" align="center" cellpadding="0" cellspacing="0">
      <tr>
        <td width="900" colspan="3"><a href="index.php"><img src="images/title.jpg" width="1000" height="121" border="0" /></a></td>
        </tr>
      <tr bgcolor="#CC9933">
        <td height="292" colspan="3">
       <form name="formcheck" id="formcheck" onsubmit="javascript:return check();" action="./reg_save.php" method="post">

          <table width="790" border="0" align="center" cellpadding="5" cellspacing="0" >
            <tr>
              <td height="55" colspan="10"><div align="center"><span class="header"><strong>REGISTRATION FORM</strong></span></div></td>
              </tr>
            <tr>
              <td width="51">Name</td>
              <td width="1">&nbsp;</td>
              <td colspan="4"><label>
                <input name="name" type="text" id="name" size="35" maxlength="100" style="width:300px" />
              </label></td>
              <td width="1">&nbsp;</td>
              <td width="146">Contact No</td>
              <td width="153"><label>
                <input type="text" name="phone" id="phone" maxlength="100" style="width:150px"/>
              </label></td>
              <td width="153"><label>
                      <input type="checkbox" name="publish" checked="false" />
                Publish</label></td>
            </tr>
            <tr>
              <td height="32">Age</td>
              <td>&nbsp;</td>
              <td colspan="4"><label>
                <input name="Regdate" type="text" id="Regdate"  style="width:300px" value="18" />
              </label></td>
              <td>&nbsp;</td>
              <td>Email ID </td>
              <td colspan="2"><label>
                <input type="text" name="email" id="email"  maxlength="100" style="width:300px"/>
              (This will be your username for login)</label></td>
            </tr>
            <tr>
              <td height="66" rowspan="3">Gender</td>
              <td rowspan="3">&nbsp;</td>
              <td width="134" rowspan="3"><table width="119">
                <tr>
                  <td width="111"><label>
                    <input name="sex" type="radio" id="sex_0" value="1" checked="checked" />
                    Male</label></td>
                    </tr>
                <tr>
                  <td><label>
                    <input type="radio" name="sex" value="2" id="sex_1" />
                    Female</label></td>
                    </tr>
              </table>                </td>
              <td height="29" colspan="3">&nbsp;</td>
              <td rowspan="3">&nbsp;</td>
              <td>Password</td>
              <td colspan="2"><label>
                <input type="password" name="password" id="password"  maxlength="100" style="width:300px"/>
              </label></td>
            </tr>
            <tr>
              <td width="84" height="34">Blood Group</td>
              <td width="1">&nbsp;</td>
              <td width="74"><select name="jumpMenu" id="jumpMenu">
                <option value="0" selected="selected">Select</option>
                <option>O+ve</option>
                <option>A+ve</option>
                <option>B+ve</option>
                <option>AB+ve</option>
                <option>O-Ve</option>
                <option>A-ve</option>
                <option>B-ve</option>
                <option>AB-ve</option>
            </select></td>
              <td>Re-Enter Password</td>
              <td colspan="2"><label>
              <input type="password" name="repass" id="repass"  maxlength="100" style="width:300px"/>
              </label></td>
            </tr>
            <tr>
              <td height="46">Weight</td>
              <td width="1">&nbsp;</td>
              <td width="74"><input name="weight" type="text" id="weight" size="5" /></td>
              <td>Address</td>
              <td colspan="2"><textarea name="address2" cols="100" id="address2" style="width:300px"></textarea></td>
            </tr>
            <tr>
              <td height="59">Class</td>
              <td>&nbsp;</td>
              <td><select name="classs" id="classs">
                <!--  <option value="0" selected="selected">Select</option> -->
                <option selected="selected" value="0">Select</option>
                <option> S1S2A </option>
                <option> S1S2B </option>
                <option> S1S2C </option>
                <option> S1S2D </option>
                <option> S1S2E </option>
                <option> S1S2F </option>
                <option> S3CS </option>
                <option> S3IT </option>
                <option> S3BT </option>
                <option> S3ME </option>
                <option> S3EE </option>
                <option> S3EC </option>
                <option> S4CS </option>
                <option> S4IT </option>
                <option> S4BT </option>
                <option> S4ME </option>
                <option> S4EE </option>
                <option> S4EC </option>
                <option> S5CS </option>
                <option> S5IT </option>
                <option> S5BT </option>
                <option> S5ME </option>
                <option> S5EE </option>
                <option> S5EC </option>
                <option> S6CS </option>
                <option> S6IT </option>
                <option> S6BT </option>
                <option> S6ME </option>
                <option> S6EE </option>
                <option> S6EC </option>
                <option> S7CS </option>
                <option> S7IT </option>
                <option> S7BT </option>
                <option> S7ME </option>
                <option> S7EE </option>
                <option> S7EC </option>
                <option> S8CS </option>
                <option> S8IT </option>
                <option> S8BT </option>
                <option> S8ME </option>
                <option> S8EE </option>
                <option> S8EC </option>
                <option> S1S2MCA </option>
                <option> S3MCA </option>
                <option> S4MCA </option>
                <option> S5MCA </option>
                <option> S6MCA </option>
                <option> Staff </option>
                <option> Alumini </option>
                <option> Others </option>

              </select></td>
              <td>Times Donated</td>
              <td>&nbsp;</td>
              <td><label>
              <input name="last" type="text" id="last" size="5" value="0" />
              </label></td>
              <td>&nbsp;</td>
              <td>District</td>
              <td colspan="2">
                <select name="address3" id="address3" style="width:300px">
                    <option selected="selected">Thiruvananthapuram</option>
                    <option>Kollam</option>
                    <option>Pathanamthitta</option>
                    <option>Alappuzha</option>
                    <option>Kottayam</option>
                    <option>Idukki</option>
                    <option>Ernakulam</option>
                    <option>Thrissur</option>
                    <option>Palakkad</option>
                    <option>Malapuram</option>
                    <option>Kozhikode</option>
                    <option>Wayanad</option>
                    <option>Kannur</option>
                    <option>Kasargod</option>
                    <option>Other</option>
                </select>
              </td>
            </tr>
            
            <tr>
              <td height="74" colspan="10"> <div align="center">
                <p><input type="submit" name="submit" id="submit" value="Submit" style="width:100px"  />                
                                    
                  <input type="reset" name="reset" id="reset" value="Reset"  style="width:100px" />
                </p>
                </div>              </td>
              </tr>
          </table>
            </form>
          </td>
        </tr>
      <tr bgcolor="#990000">
        <td height="15" colspan="3" bgcolor="#FFFFFF"><div align="center"><span class="style4 style3"><a href="privacy.xhtml">Privacy Policy</a></span></div></td>
      </tr>
      
    </table></td>
  </tr>
</table>

</body>
</html>
